pkg/collector/container.go

```diff
@@ -16,10 +16,12 @@ package collector
 
 import (
 	"fmt"
+
 	"github.com/go-logr/logr"
-	"github.com/open-telemetry/opentelemetry-operator/pkg/collector/adapters"
 	corev1 "k8s.io/api/core/v1"
 
+	"github.com/open-telemetry/opentelemetry-operator/pkg/collector/adapters"
+
```

Comment:
nit: this extra line shouldn't be there :-)

Reply:
No worries. Thought i'd ran `goimports` on it, but I guess not? Regardless, all set now.

Reply:
You might have, but if you add an extra import section explicitly, it will keep it.